engine/src/flutter/vulkan/procs/vulkan_interface.h

```diff
@@ -27,6 +27,13 @@
 
 #include <vulkan/vulkan.h>
 
+// The Vulkan headers may bring in X11 headers which define some macros that
```

Comment:
Maybe and ifdef-undef pair instead? I am pretty sure this would cause errors in TUs that dont suppress the "undefining an unknown macro".

Reply:
The language spec says that `#undef` can safely be used for a macro with no previous definition